README.md

```diff
@@ -188,7 +188,7 @@ Parameters that are enabled by default have to be explicitly disabled. These (cu
 | `cellpadding_y`                    | Set the vertical cellpadding, default is `-0.085` |
 | `frametime`                        | Display frametime next to fps text                                                    |
 | `table_columns`                    | Set the number of table columns for ImGui, defaults to 3                              |
-| `blacklist`                        | Add a program to the blacklist. e.g `blacklist=vkcube,WatchDogs2.exe`                 |
+| `blacklist`                        | Add a program to the blacklist. e.g `blacklist=vkcube + WatchDogs2.exe`                 |
```

Comment:
shouldn't be spaces between the blacklist names and the `+`

Reply:
Because of aesthetics? or functionality?  `parse_str_tokenize` is calling trim() on the string

Reply:
I'm assuming it would not work well with the env var

Reply:
Ahh ok yep makes sense